open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/ffdc/AnalyticsModelingErrorCode.java

```diff
@@ -164,7 +164,16 @@
 			"User {0} failed to create server capability {1}. Cause: {2}",
 			"The system is unable to process the request.",
 			"Review the exception to resolve the issue."),
-	
+	FAILED_UPDATE_ARTIFACT(
+			"OMAS-ANALYTICS-MODELING-027", 
+			"Failed to update analytics artifact.",
+			"The system is unable to process the request.",
+			"Review the exception to resolve the issue and repeat request to fix artifact."),
+	ILLEGAL_OPERATION(
+			"OMAS-ANALYTICS-MODELING-028", 
+			"Illegal operation {0} is called for instance {2}",
+			"Logical error in execution. The system is unable to process the request.",
+			"Report the error to administrator."),
```

Comment:
Who is the administrator and what should they do when they get this reported

Reply:
I fixed parameters. This internal error possible in case developer mistake. Only same object (defined by the same GUID) can be involved.